templates/instructions.txt

```diff
@@ -19,9 +19,9 @@ copilot app init
 copilot svc init --name {{ service.name }}
 {%- endfor %}
 
-# 3. Add the WAF AC ARN for each environment
+# 3. Add the WAF AC ARN for your web facing service
 
-Edit the waf.yml file in `copilot/environments/addons/waf.yml' and replace "replace-with-waf-acl-arn" with the WAF ACL ARN, for each environment.
+Edit the waf.yml file in `copilot/web/addons/waf.yml' and replace "replace-with-waf-acl-arn" with the WAF ACL ARN, for each environment.
```

Comment:
```suggestion
Edit `copilot/web/addons/waf.yml' and replace "replace-with-waf-acl-arn" with the WAF ACL ARN, for each environment.
```

Reply:
These comments are no longer relevant and can be removed.  But I thought the instructions file was ditched and replaced with a link to the docs?  If that's the case, then maybe the whole file can be removed?

Reply:
@lgarvey We left the instructions template as is and removed it from the `storage` command. It was left in for this command: `copilot-helper.py bootstrap instructions bootstrap.yml`. It should be updated at some point to pull in from the doc GitHub repo, so all the steps can be viewed in the CLI.

Reply:
I have removed the stuff about the web access firewall from this file. Will merge once checks pass.

If this file needs moving, we should. have a ticket for that.